### DISCUSSION TITLE
Gatsby Site Based on a Starter Theme is No Longer Workable

### DISCUSSION THREAD

**Discussion by K-Love** [gatsbyjs/gatsby#37756] [Help]
Hi!,

I started this Gatsby site a couple years ago:
https://www.chriswillburncoaching.com/

Here's the repo:
https://github.com/ChrisWillburn/chris-willburn1

I used a starter theme, located here:
https://github.com/anubhavsrivastava/gatsby-starter-spectral

It was fine for a while, but it's now so riddled with errors, I can't work on it locally, nor deploy it to Netlify. 

I've implemented everything from the error messages that I could.

Upgrading packages, or reverting them to the original version...nothing fixes it.

I'll admit it...I'm lost! Any help would be greatly appreciated.

Thanks for reading,

-Kevin

**Comment by LekoArts** [comment]
Hi!

The repository you linked is a starter, so a Gatsby project as a specific point in time that you clone. So you can change anything you'd like. Hard to help you without seeing the actual errors.
